app/views/api/v2/realms/main.json.rabl

```diff
@@ -1,3 +1,5 @@
 object @realm
 extends "api/v2/realms/base"
+extends "api/v2/smart_proxies/children_nodes"
+
 attributes :name, :realm_proxy_id, :realm_type, :created_at, :updated_at
```

Comment:
@domcleal : I saw the params, however we break compatibility if we just remove these. What is the normal procedure for this?